config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -204,6 +204,37 @@ periodics:
     testgrid-tab-name: node-kubelet-orphans
     description: "Contains all uncategorized tests, these are tests which are not marked with a [Node] tag. All node tests should be marked with [NodeFeature] or [NodeSpecialFeature] or [NodeAlphaFeature] or [NodeConformance] classification. Also skipped are [Flaky], [Benchmark], [Legacy]."
 
+- name: ci-kubernetes-node-kubelet-lock-contention
+  interval: 4h
```

Comment:
Just to understand the context, any particular reason on why we are using a 4 hour intervals? Ditto for the two timeout value at ln#217 and ln#229?

Reply:
They are duplicated, nice catch. the 4 hours interval was a standard for these jobs and there's no particular reason for this job. Any suggestion for a better value?